contracts/eth/nearprover/utils/upgrade_bridge_address.js

```diff
@@ -15,21 +16,28 @@ async function upgradeProversBridgeAddressTo(provider, proverAddress, newBridgeA
     console.log(`Initial bridge address: ${initialBridgeAddress}`);
     console.log(`Trying to upgrade bridge address to: ${newBridgeAddress}`);
 
-    const adminWallet = new ethers.Wallet(process.env.ETH_PRIVATE_KEY, provider);
+    let signer;
+    if (ledgerKeyPath != null) {
+        signer = new EthLedgerSigner(provider, ledgerKeyPath);
+    } else {
+        signer = new ethers.Wallet(process.env.ETH_PRIVATE_KEY, provider);
+    }
 
     assert.equal(
-        adminWallet.address,
+        await signer.getAddress(),
         await nearProver.admin(),
-        "The used account is not an admin of NearProver"
+        'The used account is not an admin of NearProver',
     );
 
     // Mask matches only on the latest 20 bytes (to store the address)
-    const mask = ethers.BigNumber.from("0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff");
+    const mask = ethers.BigNumber.from('0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff');
     const response = await nearProver
-        .connect(adminWallet)
+        .connect(signer)
         .adminSstoreWithMask(BRIDGE_ADDRESS_SLOT, newBridgeAddress, mask);
-    await response.wait(10).then(function(receipt) {
-        console.log(`Transaction mined: `);
+    console.log(response);
+    console.log('Waiting for tx confirmation...');
+    await response.wait(10).then(function (receipt) {
```

Comment:
Is the `10` seconds enough to get the tx confirmation ?

Reply:
@0x3bfc it is a number of confirms not seconds [ref](https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse)